packages/xstate-wallet/src/store/channel-store-entry.ts

```diff
@@ -9,18 +9,29 @@ import {
   StateVariablesWithHash,
   Outcome
 } from './types';
-import {hashState, calculateChannelId, createSignatureEntry, outcomesEqual} from './state-utils';
+import {
+  hashState,
+  calculateChannelId,
+  createSignatureEntry,
+  outcomesEqual,
+  toNitroSignedState
+} from './state-utils';
 import _ from 'lodash';
-import {BigNumber, bigNumberify} from 'ethers/utils';
+import {BigNumber, bigNumberify, Interface, defaultAbiCoder} from 'ethers/utils';
 
 import {Funding} from './store';
 import {Errors} from '.';
 import {logger} from '../logger';
+import {getVariablePart} from '@statechannels/nitro-protocol';
 export interface SignatureEntry {
   signature: string;
   signer: string;
 }
 
+// TODO: Pull out hard-coded app and store app artifacts somewhere by appDefinition
+import ConsensusAppArtifact from '@statechannels/nitro-protocol/build/contracts/ConsensusApp.json';
```

Comment:
Are we still using the consensus app? I thought we had switched to null channels?

Reply:
Well, it's actually worse than that. RPS uses RPS, Web3Torrent uses SingleAssetPayment

Reply:
Yeah. I think we could probably split that up into two chunks of work. This PR for validating null channels (which requires no external contracts) and then a PR to do something like discussed above (https://github.com/statechannels/monorepo/pull/1857#discussion_r425494473) (or you could do it all in one PR).

Either way I think this reference to ConsensusApp can go :)